doc/release/RELEASE-NOTES.md

```diff
@@ -9,9 +9,15 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * Product name: Open Supply Hub
 * Release date: November 29, 2025
 
+### Database changes
+
+#### Migrations
+* 0186_add_json_schema_to_partner_field.py - This migration adds a `json_schema` JSONField to the `PartnerField` model, allowing administrators to define JSON schemas for validating object type partner fields. The field is optional and is used to validate the structure and content of contributed data when the partner field type is `object`.
+
```

Comment:
Pls, add `Schema changes` section too. You can depict there changes from `src/django/api/models/partner_field.py`